features/naam/naam.feature

```diff
@@ -1,20 +1,18 @@
-      #language: nl
+#language: nl
 
-      @api
-      Functionaliteit: raadpleeg bewoning levert naam bewoner
+@api
+Functionaliteit: raadpleeg bewoning levert naam bewoner
 
-      Als provider van de bewoning informatie service
-      wil ik dat de naam en de geslachtsaanduiding van de bewoner wordt geleverd
-      zodat ik de volledige naam van de bewoner kan leveren aan de consumer van de Bewoning API
+Als provider van de bewoning informatie service
+wil ik dat de naam en de geslachtsaanduiding van de bewoner wordt geleverd
+zodat ik de volledige naam van de bewoner kan leveren aan de consumer van de Bewoning API
 
-      Achtergrond:
-      Gegeven adres 'A1' heeft de volgende gegevens
-      | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
-      | 0800                 | 0800000000000001                         |
+  Achtergrond:
+  Gegeven adres 'A1' heeft de volgende gegevens
+  | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
+  | 0800                 | 0800000000000001                         |
 
-  Regel: een persoon is binnen een periode bewoner van een adresseerbaar object als:
-  - de van datum van de periode valt op of na datum aanvang adreshouding van de persoon op het adresseerbaar object
-  - de tot datum van de periode valt vóór datum aanvang adreshouding van de persoon op het volgende adresseerbaar object
+  Regel: Naam wordt geleverd bij raadplegen van bewoning met type BewoningMetPeildatum en BewoningMetPeriode
 
   Scenario: naam van de bewoner wordt geleverd bij het raadplegen van bewoning met periode
       Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
```

Comment:
Omdat er in de scenario op regel 155 het leveren van adellijke titel/predicaat wordt gevalideerd, zou ik hier het `adellijke titel of predicaat (02.20)` verwijderen. Dan valideer je ook dat de naam velden worden geleverd als een persoon geen adellijke titel of predicaat heeft

Reply:
adellijke titel en predicaat gegevens zijn verwijderd uit dit scenario. adellijke titel en predicaat features zijn toegevoegd onder:  `features/naam/adellijke-titel-en-predicaat.feature`. De reden dat het in de folder `naam` staat is omdat adellijke titel een veld is in het naam object. 

Features over geslachtsaanduiding zijn ook apart gezet. Zie de folder `geslacht`, feature `geslacht.feature`. De reden dat dit in een aparte folder staat is omdat geslachtsaanduiding als aparte categorie wordt geleverd en geen onderdeel is van het naam object.